MIDAS/src/systems.cpp

```diff
@@ -154,6 +154,17 @@ DECLARE_THREAD(buzzer, RocketSystems* arg) {
     }
 }
 
+/**
+ * See \ref data_logger_thread
+ */
+DECLARE_THREAD(camera, RocketSystems* arg) {
```

Comment:
why do you tick the buzzer inside the camera thread but don't tick the camera control